.eslintrc.js

```diff
@@ -207,8 +207,15 @@ module.exports = {
     'use-isnan': 'error',
     'valid-typeof': 'off',
     'no-eq-null': 'error',
+    camelcase: ['error', { ignoreDestructuring: true, properties: 'never', allow: ['_datadog_xhr', '_dd_temp_'] }],
```

Comment:
What is the reasoning for:

- `ignoreDestructuring: true`
- `properties: 'never'`

Reply:
`ignoreDestructuring: true` can be removed.
For `properties: 'never'` there is 149 refs impacted by it. Most of the time it's when we want to serialize `RawRumViewEvent` or make expectations in spec files. 
Ideally, I would make lint exceptions for certain files, but I can't really find any patterns to cover all cases. Maybe by isolating the serialisation, we can achieve it. 
What do you think?

Here is the list of impacted files:
> /logs/src/boot/logs.spec.ts
> /logs/src/boot/logs.ts
> /rum-core/src/boot/rum.ts
> /rum-core/src/boot/rumPublicApi.spec.ts
> /rum-core/src/domain/assembly.spec.ts
> /rum-core/src/domain/assembly.ts
> /rum-core/src/domain/internalContext.spec.ts
> /rum-core/src/domain/internalContext.ts
> /rum-core/src/domain/rumEventsCollection/action/actionCollection.spec.ts
> /rum-core/src/domain/rumEventsCollection/action/actionCollection.ts
> /rum-core/src/domain/rumEventsCollection/error/errorCollection.spec.ts
> /rum-core/src/domain/rumEventsCollection/error/errorCollection.ts
> /rum-core/src/domain/rumEventsCollection/longTask/longTaskCollection.spec.ts
> /rum-core/src/domain/rumEventsCollection/longTask/longTaskCollection.ts
> /rum-core/src/domain/rumEventsCollection/resource/resourceCollection.spec.ts
> /rum-core/src/domain/rumEventsCollection/resource/resourceCollection.ts
> /rum-core/src/domain/rumEventsCollection/resource/resourceUtils.spec.ts
> /rum-core/src/domain/rumEventsCollection/resource/resourceUtils.ts
> /rum-core/src/domain/rumEventsCollection/view/viewCollection.spec.ts
> /rum-core/src/domain/rumEventsCollection/view/viewCollection.ts
> /rum-core/test/fixtures.ts
> /rum-core/test/specHelper.ts
> /rum-recorder/src/boot/recorder.spec.ts
> /rum-recorder/src/domain/record/observer.ts
> /rum-recorder/src/domain/record/record.spec.ts
> /rum-recorder/src/domain/record/record.ts
> /rum-recorder/src/domain/record/serialize.spec.ts
> /rum-recorder/src/domain/record/serialize.ts
> /rum-recorder/src/domain/segmentCollection/segment.spec.ts
> /rum-recorder/src/domain/segmentCollection/segment.ts
> /test/browsers.conf.js
> /test/e2e/scenario/recorder.scenario.ts
> /test/e2e/scenario/rum/actions.scenario.ts
> /test/e2e/scenario/rum/init.scenario.ts

Reply:
ok, let's remove the useless config and go with that.